site/_posts/2017-09-29-miqldap-to-sssd.md

```diff
@@ -0,0 +1,315 @@
+---
+title: "MiqLdap To SSSD conversion tool"
+author: jvlcek
+date: 2017-09-29
+comments: true
+published: true
+tags: instruction
```

Comment:
I've been pondering over this, I'd prefer the tag "tutorials". What do you think?

also ping @JPrause @Fryguy

Reply:
@cybette, @JPrause, @Fryguy I'm OK with setting it to whatever is agreed upon. Please let me know.